File: c /c language/while/sum2.dSYM/Contents/factorial.c
#include<stdio.h>
int main()
{
int i,a,fact=1;

printf("\n enter a=");
scanf("%d",&a);

for(i=1;i<=a;i++)
{
   
   fact=fact*i;
  

}

printf("%d",fact);

return 0;
}

File: java/factorial.java
import java.util.*;

// public class factorial
// {
//     public static void main(String[] args)
//     {
//         int fact=1,i,n=3;

//         for(i=1;i<=n;i++)
//         {
//             fact=fact*i;
//         }
//         System.out.println("factorial: " + fact);
//     }
// }

//taking input
public class factorial
{
    public static void main(String[] args)
    {
        int fact=1,i,n;

        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number to find its factorial : ");
        n = sc.nextInt();

        for(i=1;i<=n;i++)
        {
            fact=fact*i;
        }
        System.out.println("factorial: " + fact);
    }
}